flake.nix

```diff
@@ -308,6 +308,8 @@
         tuxedo-infinitybook-pro14-gen7 = import ./tuxedo/infinitybook/pro14/gen7;
         tuxedo-pulse-14-gen3 = import ./tuxedo/pulse/14/gen3;
         tuxedo-pulse-15-gen2 = import ./tuxedo/pulse/15/gen2;
+        xiaomi-fix-boot = import ./xiaomi/common/fix-boot;
```

Comment:
For I would encourage users to include their own laptop module rather than using this:

```suggestion
```